src/script/components/panel/ConversationProtocolDetails/ConversationProtocolDetails.tsx

```diff
@@ -81,14 +81,14 @@ const wrapperStyles: CSSObject = {
 
 const ConversationProtocolDetails: React.FC<ConversationProtocolDetailsProps> = ({protocol, cipherSuite}) => {
   return (
-    <>
+    <div>
```

Comment:
We need to wrap all of its content with block element - knockout components seems to be rendered as block element - taking all horizontal space. Styles were broken when rendered in react without the `div`